soh/soh/Enhancements/randomizer/location_access/overworld/gerudo_valley.cpp

```diff
@@ -41,12 +42,12 @@ void RegionTable_Init_GerudoValley() {
 
     areaTable[RR_GV_LOWER_STREAM] = Region("GV Lower Stream", "Gerudo Valley", {RA_GERUDO_VALLEY}, DAY_NIGHT_CYCLE, {}, {}, {
         //Exits
-        Entrance(RR_LAKE_HYLIA, []{return logic->IsChild || logic->HasItem(RG_BRONZE_SCALE);}),//can use cucco as child
+        Entrance(RR_LAKE_HYLIA, []{return logic->HasItem(RG_BRONZE_SCALE);}),
     });
 
     areaTable[RR_GV_GROTTO_LEDGE] = Region("GV Grotto Ledge", "Gerudo Valley", {RA_GERUDO_VALLEY}, DAY_NIGHT_CYCLE, {}, {}, {
         //Exits
-        Entrance(RR_GV_LOWER_STREAM,   []{return true;}),
+        Entrance(RR_GV_LOWER_STREAM,   []{return logic->HasItem(RG_BRONZE_SCALE) || logic->TakeDamage();}),
```

Comment:
with much effort xelivous was able to make this as child without bronze scale with a side hop

Reply:
Would the sidehop be precise enough to put it into trick territory?

Reply:
yeah, tho it's also pretty useless outside a very niche early game entrance rando situation that it's probably best to skip trick bloat & just leave a comment that it's possible